src/main/java/com/hackclub/hccore/commands/StatsCommand.java

```diff
@@ -61,11 +121,23 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
             }
         }
 
+        if (args.length == 2) {
```

Comment:
Can you use a switch statement for the args length check?

Reply:
i have not thought of that.